autosar/behavior.py

```diff
@@ -680,7 +680,7 @@ def find(self,ref):
                 return foundElem
         return None
 
-    def createRunnable(self, name, portAccess=None, symbol=None, concurrent=False, exclusiveAreas=None, modeSwitchPoint = None, adminData=None):
+    def createRunnable(self, name, portAccess=None, symbol=None, concurrent=False, exclusiveAreas=None, modeSwitchPoint = None, minStartInterval = 0, adminData=None):

```

Comment:
Well done. I will consider if it makes sense to let minStartInterval be None by default and. (Will possibly do that change later).